src/test/testFiles/valid/stackFunction.manimdsl

```diff
@@ -1,5 +1,5 @@
 fun function(x: number): Stack<number> {
-    let stack1 = new Stack<number>;
+    let stack1 = new Stack<number>();
```

Comment:
Also I think we discussed that because we're adding "()" in, we can remove the "new" keyword completely, it's not really necessary

Reply:
Sure!